server/src/main/java/org/elasticsearch/index/query/MatchQueryBuilder.java

```diff
@@ -328,27 +328,41 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
         builder.startObject(fieldName);
 
         builder.field(QUERY_FIELD.getPreferredName(), value);
-        builder.field(OPERATOR_FIELD.getPreferredName(), operator.toString());
+        if (operator != Operator.OR) {
```

Comment:
Most of the other checks run against a default value by way of a variable that's explicitly named as such; I know this default isn't likely to change but is there something similar we could do here? This is pretty nitpicky so I'm okay with it as-is if the answer is "no".